libexec/azhpc-build.sh

```diff
@@ -307,48 +331,82 @@ for storage_name in $(jq -r ".storage | keys | @tsv" $config_file 2>/dev/null);
                 mkdir -p scripts
                 status "Building script: $mount_script"
                 echo "#!/bin/bash" > $mount_script
-                echo "yum install -y nfs-utils" >> $mount_script
+                echo "yum install -y nfs-utils cifs-utils" >> $mount_script
```

Comment:
This would be better if we only installed the packages for the protocol we want to use.

Reply:
good point; will try to optimize...